server/tests/integrations/services/SessionServices/delete.integration.test.ts

```diff
@@ -0,0 +1,47 @@
+import MemberServices from '../../../../src/services/MemberServices';
+import SessionServices from '../../../../src/services/SessionServices';
+
+import { dataSource, closeDatabase, startDatabase } from '../../../../src/db';
+import { ErrorMessages } from '../../../../src/utils/enums/ErrorMessages';
+import { randomBytes } from 'crypto';
+
+describe('Delete Session integration test', () => {
+	beforeAll(async () => {
+		await startDatabase();
+	});
+
+	afterAll(async () => {
+		await closeDatabase();
+	});
+
+	beforeEach(async () => {
+		for (const entity of dataSource.entityMetadatas) {
```

Comment:
Are we forced to use "for const" loop here? Maybe some modern method exist

Reply:
idk, i just copy the Arnaud code.

Reply:
idk, i just copy the Arnaud code.